docs/make.jl

```diff
@@ -12,8 +34,17 @@ makedocs(
         #"Quick Start Guide" => "qs_guide.md",
         "Logging" => "man/logging.md",
         "Operation Model" => "man/op_problem.md",
+        "How To" => Any[
+            "Set Up Plots" => "howto/3.0_set_up_plots.md",
+            "Make Stack Plots" => "howto/3.1_make_stack_plots.md",
+            "Make Bar Plots" => "howto/3.2_make_bar_plots.md",
+            "Make Fuel Plots" => "howto/3.3_make_fuel_plots.md",
+            "Make Forecast Plots" => "howto/3.4_make_forecast_plots.md",
+            "Plot Fewer Variables" => "howto/3.5_plot_fewer_variables.md",
+            "Plot Multiple Results" => "howto/3.6_plot_multiple_results.md",
+        ],
         "Simulation Recorder" => "man/simulation_recorder.md",
-        "API" => Any["PowerSimulations" => "api/PowerSimulations.md"],
+        #  "API" => Any["PowerSimulations" => "api/PowerSimulations.md"],
```

Comment:
Why is this commented out?

Reply:
it was throwing a warning, I can uncomment